src/core/server/config/deprecation/core_deprecations.ts

```diff
@@ -103,6 +103,17 @@ const mapManifestServiceUrlDeprecation: ConfigDeprecation = (settings, fromPath,
   return settings;
 };
 
+const requestLoggingEventDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  if (has(settings, 'logging.events.request') || has(settings, 'logging.events.response')) {
+    log(
+      '"logging.events.request" and "logging.events.response" have been deprecated and will be removed ' +
+        'in 8.0. To access request/response data moving forward, please enable debug logs for the ' +
```

Comment:
super nit: We're showing the same warning for both (or either) logging events, which might cause confusion if someone's using only one of these. WDYT of 
```suggestion
        'in 8.0. To access request and/or response data moving forward, please enable debug logs for the ' +
```
?